docs/options-reference.md

```diff
@@ -18,6 +18,9 @@ Here are all the options available when configuring the module and their default
   //   { code: 'fr', iso: 'fr-FR', file: 'fr.js' },
   //   { code: 'es', iso: 'es-ES', file: 'es.js' }
   // ]
+  //   `iso` value should have:
+  //   - code of ISO 639-1
+  //   - code of ISO 639-1 and code of ISO 3166-1 alpha-2, with a hyphen
```

Comment:
Should it rather be `should have either:`?
Also maybe add an example for each?

Reply:
Thanks, I think `should have either` is better.
And I'll add example.

Reply:
I've done in b3585f1 💪